homeassistant/components/sensor/glances.py

```diff
@@ -142,6 +141,11 @@ def state(self):
                 return value['processcount']['thread']
             elif self.type == 'process_sleeping':
                 return value['processcount']['sleeping']
+            elif self.type == 'cpu_temp':
+                for sensor in value['sensors']:
```

Comment:
At least on a default installation of Fedora `sensors` is not populated. If we need additional requirement this should be mentioned somewhere.